js/viz/axes/base_axis.js

```diff
@@ -1331,6 +1333,7 @@ Axis.prototype = {
         addToArgs(this._tickInterval);
         addToArgs(this._estimatedTickInterval);
         isDefined(interval) && minArgs.push(interval);
+        isDefined(this._aggregationInterval) && minArgs.push(this._aggregationInterval);
```

Comment:
_aggregationInterval can has 'Object' type. I suggest you use 'addToArgs' function.